linter/lints/cpcps/lint_crl_has_idp_test.go

```diff
@@ -47,10 +47,9 @@ func TestCrlHasIDP(t *testing.T) {
 			wantSubStr: "IssuingDistributionPoint FullName URI MUST be present",
 		},
 		{
-			// TODO(#7296): When we're back to only including one GeneralName within
-			// the distributionPoint's FullName, change this to expect a lint.Notice.
-			name: "idp_two_uris",
-			want: lint.Pass,
+			name:       "idp_two_uris",
+			want:       lint.Notice,
```

Comment:
The [BRs](https://github.com/cabforum/servercert/blob/c4a34fe2292022e0a04ba66b5a85df75907ac2a2/docs/BR.md#712112-crl-distribution-points) state the having more than one FullName is *NOT RECOMMENDED*. Should we consider this an error instead of a notice? I understand it's not a *MUST*.

Reply:
First a subtle but important distinction: the section of the BRs you cite says that having more than one **`DistributionPoint`** is NOT RECOMMENDED, not that having more than one `fullName` is NOT RECOMMENDED. A `DistributionPoint` is defined as a structure which contains a `DistributionPointName` along with some other metadata, and it's that `DistributionPointName` which is itself a `fullName`.

Section 7.1.2.11.2 is concerned with the [CRL Distribution Points](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.13) extension in Certificates, while this lint is concerned with the [Issuing Distribution Point](https://datatracker.ietf.org/doc/html/rfc5280#section-5.2.5) extension in CRLs, which the BRs profile in [Section 7.2.2.1](https://github.com/cabforum/servercert/blob/c4a34fe2292022e0a04ba66b5a85df75907ac2a2/docs/BR.md#7221-crl-issuing-distribution-point).

You'll note that Section 7.2.2.1 has nothing to say about containing multiple `DistributionPoints`s. This is because the IDP extension can only contain one `DistributionPointName` -- it doesn't use the same sequence of `DistributionPoint`s as the CDP extension does.

So how do our IDP extensions contain more than one URI, if they can't have more than one `DistributionPointName`? Well, a `DistributionPointName` is actually a `GeneralNames`, and that is a sequence which can contain one or more URIs. There are so many layers of nesting here, it's ridiculous.

In summary:
- What this lint is looking for is a CRL with a single Issuing Distribution Point extension with a single Distribution Point Name containing more than one URI.
- What the BRs say is NOT RECOMMENDED is a Certificate with a single CRL Distribution Point extension containing more than one DistributionPoint.

In other words, its perfectly fine for a CRL to have multiple URIs within its single Distribution Point Name, so we only want to output a Notice since it's simply unexpected.